src/Stratis.Bitcoin.Features.RPC.Tests/Controller/RPCControllerTest.cs

```diff
@@ -151,6 +152,9 @@ public void CallByName_CorrectQueryString_SendsRpcRequest_ReturnsResult()
                 });
             }
             this.descriptors.Add(descriptor);
+            var values = new Dictionary<string, StringValues>();
```

Comment:
Is this needed?

Reply:
This must have shown up while fixing the conflict, not exactly sure where it came from. Removed.